doc/DeveloperGuide.md

```diff
@@ -67,6 +67,18 @@ Priority | As a ... | I want to ... | So that I can...
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+#### Use case: Rename tag
+**MSS**
+1. User request to rename tag friend to buddies
+2. AddressBook ask for confirmation for rename tage
+3. User acknowledge the confirmation
+4. AddressBook renamed friend to buddies
+
+**Extensions**
```

Comment:
In use case extensions, the numbering is use to describe scenarios that can happen, preventing the user from going to the next step.
so, this is better off being 1a.